validation.js

```diff
@@ -0,0 +1,31 @@
+function showMessage(input, message, type) {
+  const msg = input.parentNode.querySelector('small');
+  msg.innerText = message;
+  input.className = type ? 'success' : 'error';
+  return type;
+}
+
+function showError(input, message) {
+  return showMessage(input, message, false);
+}
+
+function validateEmail(input, invalidMsg) {
+  const emailRegex = /^[^A-Z\s]+\.[^A-Z\s]+$/;
+  const email = input.value.trim();
+  if (!emailRegex.test(email)) {
+    return showError(input, invalidMsg);
+  }
+  return true;
+}
+
+const form = document.querySelector('#contact-page fieldset');
+const submissionForm = document.querySelector('#contact-page form');
+const EMAIL_INVALID = 'Please enter email in lower case !';
+
+submissionForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const emailValid = validateEmail(form.elements.useremail, EMAIL_INVALID);
+  if (emailValid) {
+    form.submit();
+  }
+});
```

Comment:
- [ ] I got the error message  screenshot below from the browser console tab when I clicked on the `Get in touch` button, kindly rectify it. 👍 

<img width="940" alt="DSC_2740-removebg-preview" src="https://github.com/anshuman7negi/my-portfolio/assets/103386183/6d5d53b9-8211-4f6e-8a8e-60e4bd776ca2">